### PR TITLE
explain how to enable heapster on minikube

### PR DESCRIPTION
**Pull request by surajssd**
The docs which ask user to see if heapster is running,
this commit adds explanation of how you can enable heapster
on minikube Kubernetes cluster.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5710)

### CODE REVIEW

File: docs/tasks/configure-pod-container/assign-cpu-resource.md [hunk 1]
```diff
@@ -23,6 +23,12 @@ A few of the steps on this page require that the
 in your cluster. But if you don't have Heapster running, you can do most
 of the steps, and it won't be a problem if you skip the Heapster steps.
 
+If you are running minikube, to enable heapster run following command:
+
```
Comment (steveperry-53) [inline docs/tasks/configure-pod-container/assign-cpu-resource.md]: run following -> run the following
(Same thing in the other file)

**Comment by steveperry-53** [comment]
@surajssd One comment. Otherwise LGTM.

**Comment by surajssd** [comment]
@steveperry-53 done!

**Comment by zacharysarah** [comment]
@surajssd :wave: I made two tiny changes for wording. LGTM! :shipit:

**Comment by surajssd** [comment]
@zacharysarah np :+1:

**Pull request merged**